src/main/target/SPRACINGF3NEO/target.h

```diff
@@ -53,6 +53,10 @@
 #define USE_MAG_AK8975
 #define USE_MAG_HMC5883
 
+#ifdef TELEMETRY_IBUS
```

Comment:
Don't use the `#ifdef` here. `#undef` will be silently ignored if the define isn't defined.

Reply:
That was the idea - to undefine it, in case it was defined - just undefining is also fine.